README.md

```diff
@@ -14,6 +14,9 @@ From HACS
 4. Restart Home Assistant
 5. Go to _Settings -> Devices & Services -> Add integration -> Eaton xComfort Bridge_
 
+## Documentation
+- [Smartstikk Switch Enhancements](docs/switch.md)
+
 ## Credits
 
-This repo is a (detached) fork of [jankrib/ha-xcomfort-bridge](https://github.com/jankrib/ha-xcomfort-bridge). 
\ No newline at end of file
+This repo is a (detached) fork of [javydekoning/ha-xcomfort-bridge](https://github.com/javydekoning/ha-xcomfort-bridge). 
```

Comment:
@jankrib is the original owner of this code. Let's keep the credits there and revert this change.